packages/wonder-blocks-banner/src/components/__tests__/banner.test.js

```diff
@@ -146,4 +146,61 @@ describe("Banner", () => {
         // Assert
         expect(link).toHaveAttribute("href", "/foo");
     });
+
+    // Test kind
+
+    test("default kind displays info icon", () => {
+        // Arrange
+
+        // Act
+        render(<Banner text="" />);
+
+        // Assert
+        const icon = screen.getByTestId("banner-kind-icon");
+        expect(icon).toHaveAttribute("aria-label", "info");
+    });
+
+    test("info kind displays info icon", () => {
```

Comment:
**style suggestion (non-blocking):** You could use `it.each` to dedupe these tests.